libbeat/docs/communitybeats.asciidoc

```diff
@@ -51,6 +51,7 @@ https://github.com/consulthys/logstashbeat[logstashbeat]:: Collects data from Lo
 https://github.com/yedamao/mcqbeat[mcqbeat]:: Reads the status of queues from memcacheq.
 https://github.com/scottcrespo/mongobeat[mongobeat]:: Monitors MongoDB instances and can be configured to send multiple document types to Elasticsearch.
 https://github.com/adibendahan/mysqlbeat[mysqlbeat]:: Run any query on MySQL and send results to Elasticsearch.
+https://github.com/stanxii/cockroachbeat[cockroachbeat]:: Run any query on CockroachDB and send results to Elasticsearch.
```

Comment:
Please change the location of this item in the table so that it appears alphabetically (makes it easier for readers to find stuff). Thanks!